app_dart/bin/validate_task_ownership.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:cocoon_service/src/request_handlers/flaky_handler_utils.dart';
+import 'package:yaml/yaml.dart';
+
+/// Remote check based on flutter `repo` and the commit `ref`.
+///
+/// This currently supports `flutter/flutter` only.
+Future<List<String>> remoteCheck(String repo, String ref) async {
+  final HttpClient client = HttpClient();
+  final Uri ciYamlUrl = Uri.https('raw.githubusercontent.com', 'flutter/$repo/$ref/.ci.yaml');
+  final HttpClientRequest ciYamlRequest = await client.getUrl(ciYamlUrl);
```

Comment:
Some of the other bin scripts get ciYaml from a network request. Should we refactor this into a utility so they can all share that logic?

Reply:
Used existing utils function instead. Also updated `bing/generate_jspb.dart`.

> +1 to Chun-Heng's comment about testability. You should add an integration test that ensures this is testing the tip of tree to prevent breakages landing here

https://flutter-review.googlesource.com/c/recipes/+/16300 is ready to accept trigger from cocoon. Will create a new builder once the recipe cl is in.

Reply:
Didn't know we have integration tests here. Just created one instead.